cnxpublishing/bake.py

```diff
@@ -32,12 +32,16 @@ def _formatter_callback_factory():  # pragma: no cover
     exercise_match = settings.get('embeddables.exercise.match', None)
     exercise_token = settings.get('embeddables.exercise.token', None)
     mathml_url = settings.get('mathmlcloud.url', None)
-    memcache_server = settings.get('memcache_server', None)
+    memcache_servers = settings.get('memcache-servers')
```

Comment:
😆  #156 made efforts to eliminate dashed configuration setting names. Please make adjust this to use `_` instead.